examples/hyperparam/train.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import print_function
+
+import warnings
+
+import math
+
+import keras
+import numpy as np
+import pandas as pd
+
+import click
+
+from keras.callbacks import Callback
+from keras.models import Sequential
+from keras.layers import Dense, Lambda
+from keras.optimizers import SGD
+from sklearn.metrics import mean_squared_error, mean_absolute_error, r2_score
+from sklearn.model_selection import train_test_split
+
+import mlflow
+import mlflow.keras
+
+"""
```

Comment:
Move this comment to the top of the file so it's easier to find; this is a doc comment for the whole file, not a method or something